agent/nmap_wrapper.py

```diff
@@ -24,8 +25,14 @@ def _parse_output(xml_output: str) -> Any:
     Returns:
         dict of the scan's result.
     """
-    parsed_xml = xmltodict.parse(xml_output)
-    return parsed_xml
+    try:
+        parsed_xml: dict[str, Any] = xmltodict.parse(xml_output)
```

Comment:
```suggestion
        parsed_xml: dict[str, Any] = xmltodict.parse(xml_output, disable_entities=True)
```

I think it's better to explicitly set `disable_entities` to `True`, right now it's set to `True` by default but that might change in some future release.